### PR TITLE
6576 multiple boxes

### PR DESCRIPTION
**Pull request by h0st**

### CODE REVIEW

File: core/generator.rb [hunk 1]
```diff
@@ -442,10 +443,10 @@ def Generator.nodeDefinition(node, boxes, path, cookbook_path)
           machine = getDockerDef(cookbook_path, name, template_path, provisioned)
           copyDockerfiles(path, name, platform, platform_version)
         else
-          $out.warning 'WARNING: Configuration has not support AWS, config file or other vm provision'
+          $out.warning 'Configuration has not support AWS, config file or other vm provision!'
```
Comment (timofeyturenko) [inline core/generator.rb]: I do not understand this message (does not support?)

File: recipes/cookbooks/galera/recipes/install_galera.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@
 [
   "rsync", "sudo", "sed", 
   "coreutils", "util-linux", "curl", "grep", 
-  "findutils", "gawk", "iproute, net-tools"
+  "findutils", "gawk", "iproute", "net-tools"
```
Comment (timofeyturenko) [inline recipes/cookbooks/galera/recipes/install_galera.rb:10]: It does NOT have any connection to 6576!

File: recipes/cookbooks/mariadb/recipes/install_community.rb [hunk 3]
```diff
@@ -3,6 +3,11 @@
 include_recipe "mariadb::mdbcrepos"
 
 
+# install net-tools
```
Comment (timofeyturenko) [inline recipes/cookbooks/mariadb/recipes/install_community.rb]: again, NO connection to 6576

File: recipes/cookbooks/galera/recipes/install_galera.rb (hunk 2 above)
Comment (h0st) [inline recipes/cookbooks/galera/recipes/install_galera.rb:10]: Yes, from now I will create separate branches for every fix!

File: recipes/cookbooks/mariadb/recipes/install_community.rb (hunk 3 above)
Comment (h0st) [inline recipes/cookbooks/mariadb/recipes/install_community.rb]: It is my mistake, I dont create branch for this fix and I decided to put it in my current branch (6576).

File: core/generator.rb (hunk 1 above)
Comment (h0st) [inline core/generator.rb]: Ok, I will fix it. It means that configuration does not contain AWS config or other vm provision other then vm, aws, libvirt, docker

File: recipes/cookbooks/galera/recipes/install_galera.rb (hunk 2 above)
Comment (h0st) [inline recipes/cookbooks/galera/recipes/install_galera.rb:10]: Now it is 6619_net_tools fix, branch merged with it.

File: core/generator.rb [hunk 4]
```diff
@@ -97,9 +95,9 @@ def Generator.writeFile(name, content)
   def Generator.getVmDef(cookbook_path, name, host, boxurl, vm_mem, template_path, provisioned)
 
     if template_path
-      templatedef = "\t"+name+'.vm.synced_folder '+quote(template_path)+", "+quote("/home/vagrant/cnf_templates")
+      templatedef = "\t"+name+'.vm.synced_folder '+quote(template_path)+", "+quote('/home/vagrant/cnf_templates')
     else
-      templatedef = ""
+      templatedef = ''
```
Comment (h0st) [inline core/generator.rb:100]: Delete quiotes ' ' from commit? It is not influence to changes, just ruby code style.

**Pull request merged**